Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -210,6 +234,22 @@ bool IsPointOutsideCalibration(Common::DVec2 point, ControllerEmu::ReshapableInp
   return current_radius > input_radius * ALLOWED_ERROR;
 }
 
+void DrawVirtualNotches(QPainter& p, ControllerEmu::ReshapableInput& stick, QColor gate_brush_color)
```

Comment:
```suggestion
void DrawVirtualNotches(QPainter& p, ControllerEmu::ReshapableInput& stick, QColor gate_brush_color)
```

Reply:
What?

Reply:
This was likely a suggestion I meant to delete during review but accidentally left in. This can be disregarded.